config/configtls/go.mod

```diff
@@ -12,7 +12,7 @@ require (
 
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
-	github.com/google/go-tpm-tools v0.4.4 // indirect
+	github.com/google/go-tpm-tools v0.4.5 // indirect
```

Comment:
seems like this shouldn't be in this PR

Reply:
Good catch. It was a remnant from [this](https://github.com/open-telemetry/opentelemetry-collector/pull/12812#discussion_r2069282615) but seems to be resolved now. I've removed the change.